showcase/src/app/app.component.html

```diff
@@ -98,7 +98,7 @@ <h1>STARK</h1>
 			<a mat-list-item uiSref="demo-table" uiSrefActive="active">
 				<span matLine>Table</span>
 			</a>
-			<a mat-list-item uiSref="stark-header" uiSrefActive="active">
+			<a mat-list-item uiSref="demo-stark-header" uiSrefActive="active">
```

Comment:
Shouldn't it be just "demo-header" without the "stark prefix"? Normally everything we demo in the showcase is about Stark right?